package.json

```diff
@@ -2,12 +2,20 @@
   "name": "aemssgdevelop",
   "version": "0.0.1",
   "private": true,
+  "bin": {
+    "setup": "./bin/setup/index.js"
+  },
   "dependencies": {
-    "@adobe/aio-lib-runtime": "^7.1.0",
+    "@adobe/aio-lib-files": "^4.1.2",
```

Comment:
no need to list these dependencies separately, you can import them from `@adobe/aio-sdk`